src/ja/home.html

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Open Web Advocacy'
+metaDesc: 'オープンなウェブの未来を擁護するために世界中から集まったソフトウェアエンジニアのグループ'
+permalink: '/ja/'
+layout: 'layouts/home.njk'
+isHome: true
+---
+
+<p class="home-intro">
+  私たちは<strong>オープンなウェブの未来</strong>を擁護するために世界中から集まったソフトウェアエンジニアのグループです。規制当局、立法者、政策立案者に対して複雑な技術の詳細情報を提供することで、この業界における競争を阻害する主要な問題とその解決方法の理解をサポートしています。
+</p>
+
+<div class="highlight">
+  <h2>危機にさらされているのは何か？</h2>
+
+  <div class="group">
+    <p>アプリケーション開発の未来すべてが危機にさらされています。規制や立法による変化なくしては、普遍的で、無料かつオープンな、一度書くだけでよく、どこにでもデプロイ可能な、アプリケーション頒布とデプロイのシステムを失うリスクがあります。このシステムは、企業と消費者のコストを大幅に削減するシステムです。
+    <p>もし規制や立法による変化がなければ、資金はプロプライエタリなエコシステムに向かい、ゲートキーバーたちは重税を徴収することが可能になります。これにより、ゲートキーバーのコントロール下にロックインされ、今後何年にもわたってモバイルアプリのイノベーションが阻害されることになってしまいます。</p>
+  </div>
+</div>
+
+<div class="presentation">
+  <div class="policy-entry">
+    <h2>どこから始めればいい？</h2>
+    <p>開発者のアドホック・グループとして、私たちは<a href="/ja/apple-browser-ban">#AppleBrowserBan</a>（Appleによるブラウザーの禁止）がオープンなウェブの未来に対する大きな脅威であると確信しました。イギリスの独占規制当局である、競争・市場庁（Competition and Markets Authority、CMA）が<a href="https://www.gov.uk/cma-cases/mobile-ecosystems-market-study">モバイルエコシステムに対する市場調査</a>を立ち上げた際に、何人かのイギリスの「メンバー」が、CMAに対してブリーフィングを行うために招待されました。</p>
+
+    <p>私たちはこの調査の一環として、規制当局への提出書『<a href="/ja/walled-gardens-report/">Bringing Competition to Walled Gardens（ウォールド・ガーデンに競争をもたらす）</a>』を作成しました。</p>
+
+    <!-- <p><a href="/ja/policy/" class="action-button">About our actions</a></p> -->
+  </div>
+
+  <div class="priorities-entry">
+    <h2>優先すべきことトップ3</h2>
+
+    <dl class="counter">
+      <dt>
+        <a href="/ja/apple-browser-ban">#AppleBrowserBan</a>
+      </dt>
+      <dd><strong>Appleによるサードパーティ製ブラウザの禁止</strong>は、iOS上で行われてきた非常に反競争的な行為です。Safari/WebKitチームの資金援助を制限することで、過去10年間に渡ってイノベーションを停滞させ、モバイル上でウェブアプリが発展することを妨げてきました。</dd>
+
+      <dt>深いシステム統合</dt>
+      <dd><strong>ウェブアプリは普通のアプリになる必要があります。</strong>無料でオープンなウェブ技術で構築されたアプリには、平等な扱いと統合が必要です。閉鎖的で重税が課されたプロプライエタリなエコシステムは、いかなる優遇も受けるべきではありません。
+
+      <dt>ウェブアプリの平等性</dt>
+      <dd><strong>ゲートキーバーたちによって設置された人工的な障壁は、すべて取り除かなければなりません。</strong>ウェブアプリは、許可さえあれば、要求の厳しいユースケースに対して、より優れたプライバシーとセキュリティを備えた同等の機能を提供できます。</dd>
+    </dl>
+  </div>
+</div>
+
+<div class="get-involved">
+  <div class="who-entry">
+    <div class="group">
+      <h2>ぜひ支援してください</h2>
+      <p>イギリス、EU、日本、オーストラリア、アメリカ合衆国の規制当局は、Appleのサードパーティ製ブラウザエンジンの禁止に対して捜査を開始しており、ウェブアプリがネイティブアプリと競合できるようにするための介入を検討しています。最も重要な点は、この趣旨の文言がEUの画期的な技術法であるデジタル市場法にすでに含まれていることです。</p>
+
+      <p>これは必然の結果ではありません。過去2年間に渡って、Open Web Advocacyが規制当局や議員に対して、モバイルアプリの複占の影響を相殺できるウェブの可能性について、広範囲に説明してきた結果です。</p>
+
+      <p>私たちの擁護活動のほぼ確実な結果として、AppleはWebKitとSafariの開発に携わる多数の追加スタッフを雇用しました。この追加のリソースと規制の圧力は、ウェブアプリのプッシュ通知やアプリのバッジなどの長期間延期されていた機能の追加に直接貢献しました。WebKitのスタッフ数は秘密にされていますが、これは大幅な増加だと考えられます。</p>
+
+      <p><strong>競争、あるいは少なくともその脅威は、正しく機能するのです。</strong></p>
+
+      <p>競争を取り戻し、ユニバーサルなウェブアプリを現実のものとするために圧力をかけ続けるために、OWAにはあなたの助けが必要です。この取り組みをサポートすれば、ウェブの将来を確実に変えることができます。ウェブの発展を阻止するためにあらゆるインセンティブを持つ、信じられないほど潤沢な資金を持つゲートキーバーたちに挑戦するために、私たちは寄付に頼っています。</p>
```

Comment:
- "インセンティブ"はビジネスシーンでしか使われない印象。別の表現に変えたほうがいいかな……
```suggestion
      <p>競争を取り戻し、ユニバーサルなウェブアプリを現実のものとするための圧力をかけ続けるために、OWAにはあなたの助けが必要です。この取り組みをサポートすれば、ウェブの将来を確実に変えることができます。ウェブの発展を阻止するためにあらゆる行動を起こす、信じられないほど潤沢な資金を持つゲートキーバーたちに挑戦するために、私たちは寄付に頼っています。</p>
```